src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+    ViewData["Title"] = "Permission to share details";
+}
+
+@section Back {
+    <a href="/ProfessionalReferral/ConnectFamilyToServiceStart?id=@Model.Id&name=@Model.Name" class="govuk-back-link">Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @if (!Model.ValidationValid)
+        {
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
```

Comment:
Could you please check if we should have the `aria-labelledby` and `role`?
Looking at the [error summary page](https://design-system.service.gov.uk/components/error-summary/), the examples don't have them.

Reply:
Fixed